README.md

```diff
@@ -91,15 +96,15 @@ const { json, meta } = superjson.serialize(object);
 
 /*
 json = {
-  normal: 'string',
+  normal: "string",
   timestamp: "2020-06-20T04:56:50.293Z",
-  test: "/blitz/",
+  test: "/superjson/",
```

Comment:
We actually left this reference on purpose, to get people interested in Blitz. So if you stumbled on this, that's the desired outcome ;D

Reply:
So I'd love for this to be reverted :D

Reply:
This was brilliant, but at the same time it seems "intriguing" why it converts one word to another, it can be seen as a mistake.

Which kind of documentation documents the result of the code as being "wrong" to get attention? Don't you think it would make more sense to place a large Blitz banner on top of the documentation? **a11y is everything**.

Reply:
> If you think that reversing makes more sense, I can do it without a problem :)

Reply:
Yeah you're right, didn't see that above we also write "superjson".